readme.md

```diff
@@ -90,9 +91,9 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
   })
   ```
 
-* **Question:** When does `app.use()` get called?
+* **Question:** When does `app.use()` get called? 
 
-* **Your Answer:**
+* **Your Answer:** It is used to setup every middleware you want to use. It can be called generically or when specific paths are specified.
```

Comment:
app.use() will always be called once a request reaches that point of the route chain.